api/src/clojure/org/akvo/flow_api/unilog/unilog.clj

```diff
@@ -45,6 +45,14 @@
   {:form-id formId
    :id formInstanceId})
 
+(defn data-point-data [{:keys [id surveyId lat lon name identifier]}]
+  (cond-> {:id (str id)
```

Comment:
This is not exactly the same object that we return on the datapoint API. 

Even if the data from Unilog maybe enough, I think the promise of the sync API is that we return exactly the same data than the other API. 

I would  personally opt for just reading from DS also these objects.

Reply:
Good point, will double check what are we returning